README.md

```diff
@@ -1,96 +1,24 @@
-# GitHub Gems: Driving Open-Source Investments With Data
+# Christopher Akey
 
-Welcome to the GitHub Gems project! This project hosts a data analytics pipeline that enables smarter investment decisions by measuring the popularity of open-source repos on Github.
+What insights are you hoping to get?
 
-## Project Overview
+* Ideltify fast growing repos
+* Analyze a specific repo
 
-The goal of this project is to develop an efficient data pipeline that streamlines analytics, reduces manual effort, and enables deeper insights into the open-source ecosystem on GitHub. By leveraging modern data tools such as dbt (data build tool) and best practices such as dimensional modeling, we aim to create a scalable and reliable solution for data-driven decision-making.
+What are the most important metrics or data points you need to analyze?
 
-## Getting Started
+* Growth rate of stars
+* Growth rate of commits
+* Custom analysis
 
-To get started with the GitHub Gems project, follow these steps (click on the
-links for guides):
+How frequently do you need the data updated?
 
-### Set up your IDE
+Daily
 
-> ℹ️ Skip some steps if you're already set!
->
-> If you already have git, VSCode, and/or Python installed, just skip the corresponding step(s).
+What format would you like the data delivered in?
 
-1. [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+SQL database
 
-2. If you don't have a coding editor installed, [install VSCode](https://code.visualstudio.com/download). After that, [install the Python and Python extension](https://code.visualstudio.com/docs/languages/python#_install-python-and-the-python-extension).
+How do you currently analyze GitHub data?
```

Comment:
Let's remove this last section as it relates to our fictional client (and only keep features that users of the repo would want to know about).